administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -338,6 +338,7 @@ COM_LOCALISE_TOOLTIP_TRANSLATIONS_COMPLETE="Translation complete::<br>%1$s trans
 COM_LOCALISE_TOOLTIP_TRANSLATIONS_EDIT="Edit translation"
 COM_LOCALISE_TOOLTIP_TRANSLATIONS_EDITRAW="Edit translation using a text editor"
 COM_LOCALISE_TOOLTIP_TRANSLATIONS_ERROR="Error::In file %1$s, lines %2$s"
+COM_LOCALISE_TOOLTIP_TRANSLATIONS_ERROR_EMPTY_FILE="Error:: Empty file detected at %1$s."
```

Comment:
It can also be that the file does not exist.
Proposal:
"Error: the file %s does not exist or is empty"


<img width="928" alt="Screen Shot 2021-11-03 at 07 18 03" src="https://user-images.githubusercontent.com/869724/140016875-459778d1-5d58-4d04-ae37-3bb64d5d990c.png">